yaksh/models.py

```diff
@@ -758,6 +758,7 @@ class Course(models.Model):
     teachers = models.ManyToManyField(User, related_name='teachers')
     is_trial = models.BooleanField(default=False)
     instructions = models.TextField(default=None, null=True, blank=True)
+    view_grade = models.BooleanField(default=False)
```

Comment:
Default should be True!
Coz False will affect the existing data.

Reply:
It will not, it is a completely new field and the field is only used for Frontend changes in Templates. Migrations will handle this.

Reply:
The value of view_grade for the existing Courses will be taken as False, right?
Then the students will not be able to see the grades that they were seeing, right?

Reply:
Yes, the only place where students are currently supposed to see grades in the live version is in Fellowship Course.

Basically, this is the same as Show AnswerPaper, we have to keep the default ```False``` since this is a Result, the decision to release the grade should be the Moderator's decision.